src/shared-components/EventListCard/Modal/Delete/EventListCardDeleteModal.spec.tsx

```
@@ -0,0 +1,403 @@
+        hasExceptions: false,
+        progressLabel: '5 of 10',
+    },
+    eventDeleteModalIsOpen: true,
+    toggleDeleteModal: vi.fn(),
+    t: (key: string) => key,
+    tCommon: (key: string) => key,
+    deleteEventHandler: vi.fn(),
+};
+
+describe('EventListCardDeleteModal', () => {
+    afterEach(() => {
```

ESLint

[error] Delete `··`

(prettier/prettier)